test/ponyfill.js

```diff
@@ -33,7 +33,6 @@ describe('ponyfill unit tests', function () {
 
         var result = ponyfill(root);
 
-        expect(Symbol.observable).to.equal('Symbol(observable)');
```

Comment:
I don't think we should remove this, as if `Symbol` exists, we're expecting to have polyfilled it, but if it doesn't exist, we at least have the returned value.